core/dnsserver/server.go

```diff
@@ -287,8 +287,10 @@ func (s *Server) OnStartupComplete() {
 		return
 	}
 
-	for zone, config := range s.zones {
-		fmt.Println(zone + ":" + config.Port)
+	for zone := range s.zones {
+		// if the server is listening on a specific address let's make it visible in the log,
+		// so one can differentiate between all active listeners
+		fmt.Println(zone + " on " + s.Addr)
```

Comment:
Is it possible to keep this backwards compatible? We introduce spaces here.
Right now we have:
~~~
.:1053
~~~
Can we detect we did a `bind` and only extend it when we did?
~~~
.:1053
com.:1053 on 192.168.1.2 172.16.1.15
~~~
Typing that last line. How does this look with multiple `bind` statements or one with multiple addresses?

Reply:
At this point the Server is listening only one address (unicast or broadcast) 
NOTE:  the multiple BIND address BlocServers are broken into several Servers within dnsContext.MakeServers, which use  groupConfigsByListenAddr() to split by one address only.

Right now it looks like:
```
com. on dns://:6003
. on dns://127.0.0.1:6001
. on dns://[::1]:6001
```

I will change to get  (for the same servers), keeping  compatible with former version.

```
com.:6003
.:6001 on 127.0.0.1
.:6001 on ::1
```

Reply:
QUESTION: we have 3 kind of Servers (dns, tls, grpc) .. do we want so show the lkine of server is listening. In other word, do we want so show TLS or gRPC as the listening protocol ?